_health-care/_js/_components/_health-care-app.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { Nav } from './_hello-world.jsx';
```

Comment:
Oh okay, wasn't sure because when you import `DateInput` in `_hello-world.jsx` you don't use the curly brackets, so I was trying to figure out the difference. Is that because you use `export default DateInput`?